aws_sagemaker_studio/frameworks/pytorch_cnn_cifar10/pytorch_cnn_cifar10.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>This prediction seems really bad - 1 out of 4 correct. Any way to improve performance, or should we just let it go since the images also look very low-resolution?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>